web/src/app/pages/challenges/user-input/view.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block pageContent %}
+   <p class="govuk-body">
+        dignissim convallis aenean et tortor at risus viverra adipiscing at in tellus integer feugiat scelerisque varius morbi enim nunc faucibus a pellentesque sit amet porttitor eget dolor morbi non arcu
+    </p>
+    <form method="post" novalidate>
+        {{ govukInput({
+            label: {
+                text: "Enter your email",
+                classes: "govuk-label--l",
+                isPageHeading: true
+            },
+            id: "email",
+            name: "email",
+            value: emailValue or errors['email'].value,
```

Comment:
Hmm It doesn't look like you're passing `emailValue` in your controller, so this value will never get rendered. Which means on error user input would be lost